ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -84,6 +85,17 @@ typedef NS_ENUM(NSInteger, ORKChoiceAnswerStyle) {
     ORKChoiceAnswerStyleMultipleChoice
 } ORK_ENUM_AVAILABLE;
 
+/// An enumeration of the format styles available for scale answers.
+typedef NS_ENUM(NSInteger, ORKNumberFormattingStyle) {
+    /// The default decimal style.
+    ORKNumberFormattingStyleDefault,
+    
+    /// Percent style.
+    ORKNumberFormattingStylePercent,
+    
+    /// Currency style.
+    ORKNumberFormattingStyleCurrency
```

Comment:
Sorry didn't see your response earlier. I just committed code that adds locale properties to both scale answer types and the result.

I like the idea of providing a full number formatter that can be subclassed. However, as you pointed out, we still need to return the currency used and NSLocale seems the best way to do that. At this point I could add the read-only number formatter to the step answer formats in addition to the locale, or replace this new locale and just keep it on the answer result. What's your preference?